containers/conclave-build/Dockerfile

```diff
@@ -11,26 +11,25 @@ ENV LC_ALL en_US.UTF-8
 
 # TODO curl and apt-transport-https are only needed to install gramine, so they should be run as part of a single
 #  command and then removed.
+# Disable interactive dialogue
+# Otherwise, a message that requires user input will appear while installing Java 17
 RUN apt-get -y update && DEBIAN_FRONTEND=noninteractive apt-get install -y \
         build-essential \
         bash \
         curl \
         apt-transport-https \
-        python3-pip
+        python3-pip \
```

Comment:
Ah, just realised. Can we remove Python from `conclave-build` for now, since we're not using `conclave-build` for building Python enclaves. We'll see when we do CON-1229 if Python should be in `conclave-build` or a new image that extends it.

Reply:
Yes, let's remove this specific package.
Note that Gramine will install Python, it's a dependency (Gramine scripts are Python scripts) 
```
marco@bonifazi:~$ sudo apt-cache depends gramine
gramine
  Depends: libcjson1
  Depends: libcurl4
  Depends: libprotobuf-c1
  Depends: python3
  Depends: python3-click
  Depends: python3-cryptography
  Depends: python3-jinja2
  Depends: python3-protobuf
  Depends: python3-pyelftools
  Depends: python3-toml
  Conflicts: gramine-dcap
  Conflicts: gramine-oot
  Recommends: gramine-ratls-dcap
  Recommends: gramine-ratls-epid
```